meson_options.txt

```diff
@@ -7,3 +7,5 @@ option('x11-backend', type: 'feature', value: 'auto', description: 'Enable X11 b
 option('examples', type: 'boolean', value: true, description: 'Build example applications')
 option('icon_directory', description: 'Location used to look for cursors (default: ${datadir}/icons)', type: 'string', value: '')
 option('xdg-foreign', type: 'feature', value: 'auto', description: 'Enable xdg-foreign protocol')
+option('renderers', type: 'combo', choices: ['auto', 'gles2'], description:
```

Comment:
I think this should be `array` instead of `combo`, so that we can add more renderers later.

Reply:
Got it, yes it should be